clusters/build-clusters/02_cluster/assets/admin_cluster-admin-0_list.yaml

```diff
@@ -0,0 +1,27 @@
+kind: List
+apiVersion: v1
+items:
+- kind: Group
+  apiVersion: v1
+  metadata:
+    name: ci-admins
+  users:
+  - AlexNPavel
+  - bbguimaraes
+  - droslean
+  - hongkailiu
+  - petr-muller
+  - stevekuznetsov  
```

Comment:
Those are github logins of DPTP team.
Could you please `oc apply -f <this_file> after merge?
It is easier for debugging if DPTPers are admin of `ns` ci.

If your cluster is with other oauth provider than github, please add the above users.